app/signals/schema.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2023 Gemeente Amsterdam
+from drf_spectacular.openapi import AutoSchema
+from rest_framework import serializers
+
+
+class ErrorSerializer(serializers.Serializer):
+    """
+    Base serializer for error responses.
+    """
+    def to_internal_value(self, data):
+        """
+        Convert external representation to internal value.
+        """
+        return data
+
+    def to_representation(self, instance):
+        """
+        Convert internal value to external representation.
+        """
+        return instance
+
+    def update(self, instance, validated_data):
+        """
+        Perform update on the instance.
+        """
+        pass
+
+    def create(self, validated_data):
+        """
+        Create a new instance.
+        """
+        pass
+
+
+class ValidationErrorSerializer(ErrorSerializer):
+    """
+    Serializer for validation error responses.
+    """
+    errors = serializers.DictField(child=serializers.ListSerializer(child=serializers.CharField()))
+    non_field_errors = serializers.ListSerializer(child=serializers.CharField())
+
+
+class GenericErrorSerializer(ErrorSerializer):
+    """
+    Serializer for generic error responses.
+    """
+    detail = serializers.CharField()
+
+
+class UnauthenticatedErrorSerializer(GenericErrorSerializer):
```

Comment:
Why define these classes? Is there an intention to give them more purpose later?